pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[tool.poetry]
+name = "pydp"
+version = "0.1.0"
+description = ""
+authors = ["Chinmay Shah <chinmayshah3899@gmail.com>"]
+license = "Apache-2.0"
+
+[tool.poetry.dependencies]
+python = "^3.5"
+jupyter = "*"
+pandas = "*"
```

Comment:
Isn't pandas, jupyter supposed to be a dev dependency?

Reply:
possibly, but not in the original pipfile:
```[[source]]
name = "pypi"
url = "https://pypi.org/simple"
verify_ssl = true

[dev-packages]
setuptools = "*"
bumpversion = "==0.5.3"
pytest = "*"
black = "==19.10b0"
twine = "*"
sphinx = "*"
sphinx-rtd-theme = "*"
gcovr = "*"
coverage = "*"
mypy = "*"

[packages]
jupyter = "*"
pandas = "*"